context/app/static/js/pages/Template/Template.tsx

```diff
@@ -0,0 +1,163 @@
+
+  const { setDialogIsOpen, ...rest } = useCreateWorkspaceForm({
+    defaultName: title,
+    defaultTemplate: templateKey,
+    defaultJobType: job_type,
+    defaultResourceOptions: resource_options,
+    initialSelectedDatasets: datasets,
+  });
+
+  const getRawDatasetType = useCallback(
+    (name: string) => Object.keys(datasetTypeMap).find((key) => datasetTypeMap[key].includes(name)) ?? name,
+    [datasetTypeMap],
+  );
+
+  return (
+    <>
+      <StyledAccordion defaultExpanded={defaultExpanded}>
+        <StyledAccordionSummary expandIcon={<KeyboardArrowDownRoundedIcon className="accordion-icon" />}>
+          <Typography variant="subtitle1" color="inherit" component="h4">
+            {title}
+          </Typography>
+        </StyledAccordionSummary>
+        <AccordionDetails>
+          <Stack spacing={2}>
+            <Button
+              variant="contained"
+              sx={{ alignSelf: 'flex-end' }}
+              disabled={!isAuthenticated}
+              onClick={() => setDialogIsOpen(true)}
+            >
+              Try Sample Workspace
+            </Button>
+            <Stack component={SummaryPaper} spacing={1}>
+              <LabelledSectionText label="Description">{description}</LabelledSectionText>
+              {assay_display_name && (
+                <LabelledSectionText
+                  label="Dataset Types"
+                  iconTooltipText="Dataset types that are used in this sample workspace."
+                >
+                  <Stack spacing={1} direction="row">
+                    {assay_display_name.map((name, idx) => (
+                      <InternalLink
+                        href={`/search?raw_dataset_type_keyword-assay_display_name_keyword[${getRawDatasetType(name)}][0]=${encodeURI(name)}&entity_type[0]=Dataset`}
+                        key={name}
+                      >
+                        {idx === assay_display_name.length - 1 ? name : `${name}, `}
+                      </InternalLink>
+                    ))}
+                  </Stack>
+                </LabelledSectionText>
+              )}
+            </Stack>
+            <WorkspaceDatasetsTable datasetsUUIDs={[...datasets]} isSelectable={false} />
+          </Stack>
+        </AccordionDetails>
+      </StyledAccordion>
+      <NewWorkspaceDialogFromExample example={example} {...rest} />
+    </>
+  );
+}
+
+interface TemplatePageProps {
+  templateKey: string;
+}
+
+function Template({ templateKey }: TemplatePageProps) {
+  const { templates } = useWorkspaceTemplates();
+  const template = templates[templateKey];
+  const datasetTypeMap = useDatasetTypeMap();
+
+  if (!template) {
+    return null;
+  }
+
+  return (
+    <Stack spacing={4} marginBottom={5}>
+      <Stack spacing={2}>
+        <SummaryData
+          title={template.title}
+          entity_type="WorkspaceTemplate"
+          entityTypeDisplay="Workspace Template"
+          status=""
+          mapped_data_access_level=""
```

Comment:
Should these just be undefined?

Reply:
Seconded; these props are currently marked as required, but it should be pretty straightforward to edit that component to add the appropriate checks to handle `undefined` cases.